app/frontend/components/domains/users/invite-screen.tsx

```diff
@@ -0,0 +1,234 @@
+import {
+  Box,
+  Button,
+  Container,
+  Flex,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  InputGroup,
+  Select,
+  Tag,
+  Text,
+} from "@chakra-ui/react"
+import { faCircleCheck, faCircleExclamation, faClose, faPaperPlane, faPlus } from "@fortawesome/free-solid-svg-icons"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { observer } from "mobx-react-lite"
+import React from "react"
+import { Controller, FormProvider, useFieldArray, useForm, useFormContext } from "react-hook-form"
+import { useTranslation } from "react-i18next"
+import { useNavigate } from "react-router-dom"
+import { EMAIL_REGEX } from "../../../constants"
+import { useJurisdiction } from "../../../hooks/resources/use-jurisdiction"
+import { useMst } from "../../../setup/root"
+import { EUserRoles } from "../../../types/enums"
+import { SharedSpinner } from "../../shared/base/shared-spinner"
+import { RouterLink } from "../../shared/navigation/router-link"
+
+interface IInviteScreenProps {}
+
+type TFormData = {
+  users: { firstName?: string; lastName?: string; email?: string; role: EUserRoles; jurisdictionId: string }[]
+}
+
+export const InviteScreen = observer(({}: IInviteScreenProps) => {
+  const { t } = useTranslation()
+  const { jurisdiction, error } = useJurisdiction()
+  const {
+    userStore: { invite },
+  } = useMst()
+
+  const defaultUserValues = {
+    role: EUserRoles.reviewManager,
+    jurisdictionId: jurisdiction?.id,
```

Comment:
if jurisdiction has not loaded/ slow to load, this might be empty. Not 100% sure, but when jurisdiction loads after a while, this might not auto update the default value. Maybe double check this. If it does not load, will need a useEffect to manually update the value